### PR TITLE
Add back signature verification for attestations

### PR DESCRIPTION
**Pull request by 0xKiwi**
This was removed due to focuses on completing chain sync first, solved by making sure it was up to date with the spec.

Depends on #1997

### CODE REVIEW

**Comment by prestonvanloon** [comment]
Please do not submit until after #1967

**Comment by prestonvanloon** [comment]
Please add a PR description that outlines your changes

File: beacon-chain/core/blocks/block_operations.go [hunk 1]
```diff
@@ -531,20 +577,9 @@ func verifyAttestation(beaconState *pb.BeaconState, att *pb.Attestation, verifyS
...
-		//   ],
-		//   message_hash=[
-		//   hash_tree_root(AttestationDataAndCustodyBit(data=attestation.data, custody_bit=0b0)),
-		//   hash_tree_root(AttestationDataAndCustodyBit(data=attestation.data, custody_bit=0b1)),
-		//   ],
-		//   signature=attestation.aggregate_signature,
-		//   domain=get_domain(state.fork, slot_to_epoch(attestation.data.slot), DOMAIN_ATTESTATION),
-		// )
-		return nil
+		if err := verifyAttestationSig(beaconState, att); err != nil {
```
Comment (prestonvanloon) [inline beacon-chain/core/blocks/block_operations.go]: `return verifyAttestationSig`

**Comment by prestonvanloon** [comment]
What was fixed since #1908?

**Comment by IvanTheGreatDev** [comment]
> What was fixed since #1908?

We needed a `state.slot + 1` in the verifyAttestations

**Pull request closed without merging**